src/main/java/com/flowingcode/vaadin/addons/errorwindow/ErrorWindow.java

```diff
@@ -68,29 +76,60 @@ public class ErrorWindow extends Dialog {
 
   private Registration attachListenerRegistration;
 
+  /**
+   * Constructs and initializes an ErrorWindow object with the supplied cause and default production mode and i18n.
+   * @param cause: The cause of the error
```

Comment:
There should be a blank line between first sentence and param description. Same happens in all javadocs of this class.